src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,91 @@
+package duke.task;
+import java.time.format.DateTimeFormatter;
+import java.util.LinkedList;
+/**
+ * Represents a task. A <code>task</code> object has two
+ * fields, which are the task name and done-status.
+ * And it will be the super class for deadline, event and todo.
+ */
+public class Task {
+    protected String taskName;
+    protected boolean isDone;
+    protected final static DateTimeFormatter DF1 = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm");
+    protected final static DateTimeFormatter DF2 = DateTimeFormatter.ofPattern("MMM dd yyyy HH:mm");
```

Comment:
DF1 and DF2 can be more interpretable names. "inputFormat" and "outputFormat" maybe?

Reply:
I agree with this, great suggestions!

Reply:
Agree as well.